curve_apps/trend_lines/params.py

```diff
@@ -83,11 +83,15 @@ class DetectionParameters(BaseModel):
     """
     Detection parameters expected by the ui.json file format.
 
+    :param azimuth: Azimuth of the path.
+    :param azimuth_tol: Tolerance for the azimuth of the path.
+    :param damping: Damping factor between [0, 1] for the path roughness.
     :param min_edges: Minimum number of points in a curve.
     :param max_distance: Maximum distance between points in a curve.
-    :param damping: Damping factor between [0, 1] for the path roughness.
     """
 
+    azimuth: Optional[float] = None
+    azimuth_tol: Optional[float] = None
```

Comment:
Hey... When I try to use Optional pre-commit converts to float | None (think it's the pyupgrade hook).  Did you have to do something to get around it?

Reply:
Yep, as discussed you need to exclude the file in the pre-commit-config